### PR TITLE
intialize the Client stuff in FSClient

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Initialize the Client class inside FSClient like LocalClient and RemoteClient do.  This will allow for self.utils to be available.

### What issues does this PR fix or reference?
#38836

### Tests written?
No

### CODE REVIEW

File: salt/fileclient.py [hunk 1]
```diff
@@ -1356,6 +1356,7 @@ class FSClient(RemoteClient):
     the FSChan object
     '''
     def __init__(self, opts):  # pylint: disable=W0231
+        Client.__init__(self, opts)  # pylint: disable=W0233
```
Comment (gtmanfred) [inline salt/fileclient.py]: preference here?

Should i just ignore the pylint, or add an option to pass to RemoteClient that doesn't do the extra initialization, but still initializes Client?

**Review by cachedout — COMMENTED** [review]
Hrm. This doesn't seem right. It seems like we should be calling `super` here and letting RemoteClient do the work instead of duplicating this. Maybe @thatch45 can comment on why the other classes were originally written this way?

**Comment by thatch45** [comment]
This is the right approach since we don't want to call all of the components of remote client. So we want all the attrs of remote client, but we don't want to set up the same interfaces.
So this is the correct approach

**Comment by MasterNayru** [comment]
Just a heads up that PR #39589 was made to solve the same issue. It's likely that @gtmanfred has a better quality fix for this, but just flagging that my addition can probably be removed if this is accepted.

**Comment by gtmanfred** [comment]
I have updated my PR to remove the utils loading, and also to move the self.opts assignment to use the one from Client()

**Pull request merged**